packages/perseus-editor/src/item-extras-editor.tsx

```diff
@@ -43,75 +47,69 @@ class ItemExtrasEditor extends React.Component<Props> {
             <div className="perseus-answer-editor">
                 <div className="perseus-answer-options">
                     <ItemExtraCheckbox
-                        label="Show calculator:"
+                        label="Show calculator"
                         infoTip="Use the calculator when completing difficult calculations is NOT the intent of the question. DON’T use the calculator when testing the student’s ability to complete different types of computations."
                         checked={this.props.calculator}
                         onChange={(e) => {
```

Comment:
nit (probably out of scope here): I'd prefer if this (and other instances below) were named something clearer, like `newValue` or `newCheckedValue` or `newCheckedState` or something instead of `e`

Reply:
I agree. I've updated it to match what the WB checkbox uses: `newCheckedState`